content/docs/user-guide/gems/development/creating/_index.md

```diff
@@ -45,3 +45,18 @@ Your `CMakeLists.txt` file is like any other CMake file. When you create it, kee
   This will be the active target during the invocation of your Gem's `CMakeLists.txt`.
 * You can use the functions available in the core O3DE build system. See the contents of the `cmake` directory in source.
 * Avoid the use of `file(DOWNLOAD ...)`. The package system of O3DE is a robust replacement, and should be used instead.
+
+# Creating an Asset-Only Gem
+
+When a gem is created using the o3de `o3de create-gem -gp <path to create gem at>` command, by default it creates a Gem that can used for packaging code and assets.
+This Gem is actually created using the "DefaultGem" template.
+There is a way to create a gem that is only meant for providing asset only and not code.
+This can can be done by specifying the "AssetGem" template through the `create-gem` command
+```cmd
+o3de create-gem -gp <path to create gem at> -tn AssetGem
+# or
+o3de create-gem -gp <path to create gem at> -tp <engine-root>\Templates\AssetGem
+```
+
```

Comment:
- Reworded your paragraph to use an active voice, instead of a passive voice. This complies with the O3DE Style Guide. ("When you create a Gem" (active) vs. "When a gem is created" (passive))
- Other minor suggestions improve reading flow. 
- Gem is a proper nound and must be capitalized. 

```suggestion
When you create a Gem without a specified template by using the command `o3de create-gem -gp <path to create a gem at>`, it's created from the "DefaultGem" template. This creates a Gem for packaging both code and assets. You can also create a Gem that provides only assets by specifying the "AssetGem" template through the `create-gem` command:

```cmd
o3de create-gem --gem-path <path to create gem at> --template-name AssetGem
# or
o3de create-gem --gem-path <path to create gem at> --template-path <engine-root>\Templates\AssetGem
```

```

Reply:
Thanks for the edit, I'll commit that suggestion.